rpm/python-cryptography.spec

```diff
@@ -0,0 +1,151 @@
+%bcond_without python3
+%bcond_with python2
+
+%bcond_with tests
+
+%{!?python3_pkgversion:%global python3_pkgversion 3}
```

Comment:
What's the point of this macro? Can't we just use 3 there? I'm not sure I understand why this is useful.